lxd/containers_get.go

```diff
@@ -43,9 +44,67 @@ func urlInstanceTypeDetect(r *http.Request) (instancetype.Type, error) {
 	return instancetype.Any, nil
 }
 
+// Pull out the Instance in order to filter
```

Comment:
Should be fine to have `evaluateFieldInstance` take a `api.Instance` instead of `*api.Instance` and drop this function.
Effectively making it up to the caller to pass the subset of `InstanceFull` if needed.